pkg/controller/autoscaler/horizontalpodautoscaler_controller.go

```diff
@@ -17,21 +17,16 @@ limitations under the License.
 package autoscalercontroller
```

Comment:
Additionally, IMO there should either be a "horizontal" directory with this being a "pod.go" file therein, or a "pod" directory with this being a "horizontal.go" file. "horizontalpodautoscaler_controller.go" is not readable.